FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -68,12 +73,32 @@ export function SignUpFormContentComponent(props) {
     const otherOption = name === 'yurosoba' ? 'fop' : 'yurosoba';
     setValue(otherOption, false);
     if (!getValues('yurosoba') && !getValues('fop')) {
-      setError('businessEntity', { type: 'manual', message: errorMessageTemplates.required });
+      setError('businessEntity', { type: 'manual', message: errorMessageTemplates.requiredRepresentative });
     } else {
       clearErrors('businessEntity');
     }
   };
 
+  const validateNameSurname = (value) => {
+    const allowedSymbolsPattern = /^[a-zA-Zа-щюяьА-ЩЮЯЬїЇіІєЄґҐ'\s]+$/;
+    const letterCount = (value.match(/[a-zA-Zа-щюяьА-ЩЮЯЬїЇіІєЄґҐ]/g) || []).length;
+    if (!allowedSymbolsPattern.test(value)) {
+      return errorMessageTemplates.notAllowedSymbols;
+    }
+    if (letterCount < 2) {
+      return errorMessageTemplates.nameSurnameFieldLength;
+    }
+    return true;
```

Comment:
The function returns either a string or boolean, which are different types. It would be better to always return the same type for easier handling of the function's return value

Reply:
I agree with you, that it would be better to return the same data type. But in this case I might just return an empty string instead of true, and I'm not sure if it is correct. 
If it correct, I'll replace true with an empty string.